src/components/Navigation/index.js

```diff
@@ -11,9 +11,7 @@ const Navigation = () => {
       <nav style={{ display: 'flex', justifyContent: 'flex-end' }}>
         <p
           onClick={() => {
-            localStorage.removeItem('userId')
-            localStorage.removeItem('token')
-            navigate('../signin')
+            handleLogout()
```

Comment:
This doesn't need to be an anonymous function. Simply pass `handleLogout` as the prop value.